firmware/application/external/protoview/ui_protoview.hpp

```diff
@@ -103,6 +103,10 @@ class ProtoView : public View {
         {screen_width - 12 * 8, 1 * 16, 96, 24},
         LanguageHelper::currentMessages[LANG_RESET]};
```

Comment:
I don't know what LanguageHelper::currentMessages is for but feel free to request change if i need to add pause and continue label/english/enum there and use them too.